lms/static/scripts/file_picker/components/component.js

```diff
@@ -27,7 +26,8 @@ export default class Component {
         const comp = components[index];
         // handle rendering of strings and other primitive values
         if(comp && typeof comp !== 'object') {
-          return `${part}${comp}`;
+          // ensure that all string values not explicitly returned from component are sanitized
+          return `${part}${_.escape(`${comp}`)}`;
```

Comment:
I initially wondered why `comp` was enclosed in a template string here. Presumably this is to stringify it if it is not already a string, ie. the same as `String(comp)`?

Reply:
Yeah, its just a short hand way to convert something to a string.